GabrieleMeoni/END2END
# Add mcc

Solving issue #9 

* Added MCC metric 
* Added both BEST EVAL/ BEST MCC results on log
* Added a flag to switch between best-eval or best-mcc to select the model over the various iterations
* Update runscripts to include flag, use efficientnet-lite0 as baseline. Removed erroneous Upsampling factor of 5 on eval

**N.B** How to test it:

* Edit the $USE_MCC_FOR_BEST flag in `run_train.sh` to use BEST EVAL to select model
* Launch training once and check the log to be updated correctly and the model is selected according to eval metrics
* Edit the $USE_MCC_FOR_BEST flag in `run_train.sh` to use BEST MCC to select model
* Launch training once and check the log to be updated correctly and the model is selected according to mcc metrics
